changelog.d/7006.feature

```diff
@@ -0,0 +1 @@
+Extend the `web_client_location` option to accept absolute URL which causes user to be redirected. Print warning when running web client on the same hostname as homeserver.
```

Comment:
```suggestion
Extend the `web_client_location` option to accept an absolute URL to use as a redirect. Adds a warning when running the web client on the same hostname as homeserver.
```

Also feel free to include yourself as the contributor ("Contributed by <your name>") if you'd like!